components/LinkButton.jsx

```diff
@@ -8,6 +8,8 @@ const LinkButton = ({
   href,
   light,
   large,
+  medium = true,
+  small,
```

Comment:
oh true...this would be a good spot for typescript so we could have the enum/required props, but maybe for now this could just be `large ? "b1 h-16" : small ?  "b3 h-10" : "b3 h-12"` to just avoid the `medium` size altogether?

Reply:
ya i think that's a smart+clean solution

Reply:
updated this to use a string literal in [e3769f0](https://github.com/oakstudios/joinmastodon/pull/32/commits/e3769f08484f02e1b35ec290fd7e49acd05c77ef)